run_docker

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+docker run -v $PWD:/src --user root -e NB_UID=$UID -e NB_GID+$GID -it registry.hub.docker.com/mdboom/pyodide:0.1.1 /bin/bash
```

Comment:
Also I might have been cleaner to put this file under `bin/` but then I agree it's harder to make `-v $PWD:/src` work in a reliable way..

Reply:
> Wouldn't mdboom/pyodide:0.1.1 be enough?

It doesn't seem to be.  Does it work for you?

> Maybe create an org iodide / pyodide at https://hub.docker.com and put the image there?

Makes sense.  I wasn't aware you could do that.

> Maybe run with --rm option to avoid keeping images around after the command exists.

Sure.

> Also it might have been cleaner to put this file under bin/ but then I agree it's harder to make -v $PWD:/src work in a reliable way..

Yes.  I also think it's a purely developer tool and would never be "installed".